src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -221,6 +221,17 @@ public void DirectoryLongerThanMaxPathAsPath_ThrowsPathTooLongException()
 
         [Fact]
         [PlatformSpecific(PlatformID.Windows)]
+        public void ExtendedDirectoryLongerThanLegacyMaxPathSucceeds()
+        {
+            var paths = IOInputs.GetPathsLongerThanMaxPath(useExtendedSyntax: true, includeExtendedMaxPath: false);
+            Assert.All(paths, (path) =>
+            {
+                Assert.NotNull(Create(path));
```

Comment:
There's nothing else to validate about the created DirectoryInfo beyond that it's not null?